io_antarctica_scene/blender_manifest.toml

```diff
@@ -0,0 +1,21 @@
+schema_version = "1.0.0"
+
+id = "io_antarctica_scene"
+version = "4.0.0"
+name = "SuperTuxKart Exporter Tools"
+tagline = "Export items to SuperTuxKart (karts/tracks/materials)"
+maintainer = "Jean-Manuel Clemencon, Joerg Henrichs, Marianne Gagnon, Richard Qian"
```

Comment:
It doesn't seem correct to put as maintainers here people who haven't been active around STK for years.